obs2ioda-v2/README.md

```diff
@@ -14,35 +23,45 @@ If you have an environment preconfigured for `mpas-jedi`, simply source that env
    ```bash
    git clone https://github.com/NCAR/obs2ioda.git <OBS2IODA_ROOT_DIR>
    ```
-2. Create a new directory `build` and navigate into it:
-   ```bash
-   mkdir build && cd build
-   ```
-3. Locate the NCEP BUFR library by executing the following command in the `NCEP BUFR` library's build directory:
+1. Create a new directory `<OBS2IODA_BUILD_DIR>` and navigate into it:
    ```bash
-   find . -name *libbufr*
+    mkdir <OBS2IODA_BUILD_DIR> && cd <OBS2IODA_BUILD_DIR>
    ```
-4. Next, run CMake to configure the build. Specify the `CMAKE_BUILD_TYPE` option to set the build type. Currently, the supported types are `Release`, `RelWithDebInfo`, and `Debug`. Don't forget to include the path to the NCEP BUFR library:
+1. Set the variable OBS2IODA_CMAKE_ARGS to include the `obs2ioda` root directory and the absolute path of the `NCEP BUFR` library:
+    ```bash
+    OBS2IODA_CMAKE_ARGS="<OBS2IODA_ROOT_DIR> -DNCEP_BUFR_LIB=<NCEP_BUFR_LIB_PATH>"
```

Comment:
Is putting `OBS2IODA_ROOT_DIR` in the `OBS2IODA_CMAKE_ARGS` correct? 
The cmake command below explicitly provides `OBS2IODA_ROOT_DIR`:
 `cmake <OBS2IODA_ROOT_DIR> ${OBS2IODA_CMAKE_ARGS}`

Reply:
No it is not...good catch!